src/sentry/integrations/jira/integration.py

```diff
@@ -789,7 +789,14 @@ def get_create_issue_config(self, group: Group | None, user: RpcUser, **kwargs):
         project_id = params.get("project", defaults.get("project"))
         client = self.get_client()
         try:
-            jira_projects = client.get_projects_list()
+            if features.has(
+                "organizations:jira-paginated-projects", group.organization, actor=user
+            ):
+                jira_projects = client.get_projects_paginated(
+                    {"maxResults": MAX_PER_PROJECT_QUERIES}
```

Comment:
limit is 10 projects

Reply:
nit: since this is assigning the same variable for both conditions, might be simpler to read with a ternary; jira_projects = A if cond else B.